src/openms/source/CHEMISTRY/Residue.cpp

```diff
@@ -58,30 +58,39 @@ namespace OpenMS
   }
 
   Residue::Residue(const String& name,
-                   const String& three_letter_code,
-                   const String& one_letter_code,
-                   const EmpiricalFormula& formula) :
+            const String& three_letter_code,
+            const String& one_letter_code,
+            EmpiricalFormula formula,
+            double pka,
+            double pkb,
+            double pkc,
+            double gb_sc,
+            double gb_bb_l,
+            double gb_bb_r,
+            set<String> synonyms):
```

Comment:
either pass by const& or move into member

Reply:
Not possible because it has a default

Reply:
the default does not prevent any of those two options.